.gitignore

```diff
@@ -3,6 +3,7 @@
 .vscode/
 .vagrant/
 .env
+tslint.json
```

Comment:
Theres no reason to ignore the tslint file.
It basically will never change. the imported config will.